### DISCUSSION TITLE
Convenient docker image

### DISCUSSION THREAD

**Discussion by Jaxkr** [envault/envault#1] [Ideas]
**Is your feature request related to a problem? Please describe it**
I just purchased your software because I had been considering building this myself for my team. I have not installed a PHP framework in years and having to manually setup/configure/secure a database is a PITA.

**Describe the solution you'd like**
A single docker image that I can just run with one command that installs a working copy of envault.

**Comment by danharrin** [comment]
Hi, thanks for your suggestion. This is on the roadmap :) I will keep you updated here as and when we make progress.

**Comment by Jaxkr** [comment]
Thanks. Could we maybe get an install script in the mean-time? Or at least some easier copy-pastable instructions?

I haven't touched PHP since 2013 and "Set up a queue worker." is really unhelpful. I don't want to spend like an hour plus figuring our how to set up, configure, and secure a redis worker. Would really appreciate an installation script. Especially for software like yours, which saves maybe 20 seconds every week, installation shouldn't be this hard.

Looking forward to using the product once I get it working though.

**Comment by danharrin** [comment]
Hi @Jaxkr, I completely understand your concern here. In the meantime, please check out our guide on [setting Envault up with Laravel Forge](https://docs.envault.dev/article/15-setting-up-envault-on-laravel-forge), which is probably the easiest quickest way to set up at the moment. Queue workers and scheduled jobs, for example, can be configured through the Forge dashboard. Last time I checked, they have a 5 day free trial on?

I will keep you updated on the Docker image. Please let me know if you need anything else.

**Comment by bencehornak** [comment]
Hi,
I have a working proof-of-concept Dockerfile, feel free to use it as an inspiration:
https://github.com/carmino-app/envault-docker
